README.md

```diff
@@ -34,7 +34,7 @@ start by looking at [ADOPTERS.md](ADOPTERS.md).
 
 ## Community discussion
 
-The [Github discussion forum](https://github.com/containers/bootc/discussions) is enabled.
+The [GitHub discussion forum](https://github.com/containers/bootc/discussions) is enabled.
```

Comment:
can be a followup, but it seems we should do another pass to scrub `containers/bootc` in favor of `bootc-dev/bootc`

Reply:
Added https://github.com/bootc-dev/bootc/pull/1325 for followup